spec/unit/property_spec.rb

```diff
@@ -708,10 +709,10 @@ def blah
           expect(resource.x).to eq "hi1"
           expect(Namer.current_index).to eq 1
         end
-        it "when x is retrieved, coercion is run each time" do
+        it "when x is retrieved, coercion is run exactly once" do
           expect(resource.x).to eq "101"
-          expect(resource.x).to eq "102"
-          expect(Namer.current_index).to eq 2
+          expect(resource.x).to eq "101"
+          expect(Namer.current_index).to eq 1
```

Comment:
This is one of the weirdest tests I've ever seen in the codebase and I'm unsure of what its purpose is, or if it was just sort of documentation-through-code of an edge condition.

Reply:
we are changing the behavior here.... which this behavior seems better.... but like this will break SOMETHING. We'll need to document that.

Reply:
I literally have no idea what this would ever break though or why anyone would ever have done this.